docs/index.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: Docs
 permalink: /docs/
 ---
-Helmet is a collection of 14 middleware to help set some security headers.
+Helmet is a collection of 14 middleware functions to help set some HTTP response headers.
```

Comment:
We could also say "...set some security related HTTP response headers." But that might just be a matter of personal preference. I'd say it's up to you. This looks good.